docs/app/Navbar.tsx

```
@@ -0,0 +1,385 @@
+                            target={"_blank"}
+                            onClick={() => {
+                                setExpanded(false);
+                            }}
+                        >
+                            <svg
+                                fill={"currentColor"}
+                                height={"20"}
+                                viewBox={"0 0 24 24"}
+                                width={"20"}
+                                xmlns={"http://www.w3.org/2000/svg"}
+                            >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)